examples/co2_and_tracer_analysis.py

```diff
@@ -86,47 +54,40 @@ def determine_tracer(img: np.ndarray, base: np.ndarray) -> np.ndarray:
     # Rescale image to physically meaningful range [0,1]
     diff = skimage.exposure.rescale_intensity(diff)
 
-    return diff
+    return daria.Image(diff, base.metadata)
 
 
-# !----- Main routine for co2 analysis
+# Define curvature correction object, initiated with config file
+# (which can be created the workflow presented in the Jupyter notebook
+# examples/notebooks/curvature_correction_walkthrough.ipynb).
+curvature_correction = daria.CurvatureCorrection(
+    config_source=f"{os.path.dirname(__file__)}/images/config.json", width=2.8, height=1.5
+)
 
-# Read image and preprocess baseline image
-baseline = cv2.imread(str(Path(f"{os.path.dirname(__file__)}/images/co2_0.jpg")))
-baseline = preprocessing(baseline)
+# Define color correction object
+roi_cc = (slice(0, 600), slice(0, 700))
+color_correction = daria.ColorCorrection(ROI = roi_cc)
 
-# Read and preprocess test image
-co2_image = cv2.imread(str(Path(f"{os.path.dirname(__file__)}/images/co2_2.jpg")))
-co2_image = preprocessing(co2_image)
+# !----- Main routine for co2 analysis
+
+# Read baseline and co2 image and correct color and curvature
+baseline_co2 = daria.Image(f"{os.path.dirname(__file__)}/images/co2_0.jpg", curvature_correction=curvature_correction, color_correction= color_correction)
+co2_image = daria.Image(f"{os.path.dirname(__file__)}/images/co2_2.jpg", curvature_correction=curvature_correction, color_correction= color_correction )
 
 # Determine co2
-co2 = determine_tracer(co2_image, baseline)
+co2 = determine_tracer(co2_image, baseline_co2)
 
-# Plot
-plt.imshow(co2)
-plt.show(block = False)
-# Pause longer if it is desired to keep the images on the screen
-plt.pause(3)
-plt.close()
+# Plot change 3 to larger number if it is desired to keep the images on the screen
+co2.plt_show(3)
```

Comment:
This was actully just introduced to make the pytest examples test conclude. There might be some way to circumvent it though.

Reply:
I see! Indeed, best would be something like having a flag deciding whether plots are shown with a break or without, and if the flag could be controlled from inside, but also from outside via pytest - I am not sure if something like this would be  possible...